content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -654,7 +654,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `MemoryManager`: Allows setting memory affinity for a container based on
   NUMA topology.
 - `MemoryQoS`: Enable memory protection and usage throttle on pod / container using
-  cgroup v2 memory controller.
+  cgroup v2 memory controller. Read more about [Quality of Service classes for Pods](/docs/concepts/workloads/pods/pod-qos/)
```

Comment:
```suggestion
  cgroup v2 memory controller.
```
I actually think that that link is misleading. For beta, we should rewrite and possible retitle the page about Pod QoS, because there is now more than one QoS concept in play.

Reply:
Updated.